doc/sphinx/addendum/program.rst

```diff
@@ -158,34 +158,33 @@ prove some goals to construct the final definitions.
 Program Definition
 ~~~~~~~~~~~~~~~~~~
 
-.. cmd:: Program Definition @ident := @term
-
-   This command types the value term in Russell and generates proof
+   A :cmd:`Definition` command with the :attr:`program` attribute types
+   the value term in Russell and generates proof
    obligations. Once solved using the commands shown below, it binds the
    final |Coq| term to the name :n:`@ident` in the environment.
 
    .. exn:: @ident already exists.
       :name: @ident already exists. (Program Definition)
       :undocumented:
 
-   .. cmdv:: Program Definition @ident : @type := @term
+   :n:`Program Definition @ident : @type := @term`
 
-      It interprets the type :n:`@type`, potentially generating proof
-      obligations to be resolved. Once done with them, we have a |Coq|
-      type :n:`@type__0`. It then elaborates the preterm :n:`@term` into a |Coq|
-      term :n:`@term__0`, checking that the type of :n:`@term__0` is coercible to
-      :n:`@type__0`, and registers :n:`@ident` as being of type :n:`@type__0` once the
-      set of obligations generated during the interpretation of :n:`@term__0`
-      and the aforementioned coercion derivation are solved.
+   Interprets the type :n:`@type`, potentially generating proof
+   obligations to be resolved. Once done with them, we have a |Coq|
+   type :n:`@type__0`. It then elaborates the preterm :n:`@term` into a |Coq|
+   term :n:`@term__0`, checking that the type of :n:`@term__0` is coercible to
+   :n:`@type__0`, and registers :n:`@ident` as being of type :n:`@type__0` once the
+   set of obligations generated during the interpretation of :n:`@term__0`
+   and the aforementioned coercion derivation are solved.
 
-      .. exn:: In environment … the term: @term does not have type @type. Actually, it has type ...
-         :undocumented:
+    .. exn:: In environment … the term: @term does not have type @type. Actually, it has type ...
+       :undocumented:
```

Comment:
This error message does not actually exists. There is no Program-specific error message in this case, so let's just remove this as well.